pkg/virt-handler/options.go

```diff
@@ -54,23 +54,34 @@ func virtualMachineOptionsWithNetworkConfig(
 	capabilities *api.Capabilities,
 	disksInfo map[string]*containerdisk.DiskInfo,
 	clusterConfig *virtconfig.ClusterConfig,
+	vmiSpecIfaces []v1.Interface,
 ) (*cmdv1.VirtualMachineOptions, error) {
 	options := virtualMachineOptions(smbios, period, preallocatedVolumes, capabilities, disksInfo, clusterConfig)
 	if clusterConfig == nil {
 		return options, nil
 	}
+
+	domainAttachmentByPluginName := make(map[string]string)
```

Comment:
Consider using `map[string]string{}` instead of `make`.

Reply:
done. But why?

Reply:
With `make` you can define a capacity, something you do not need here.
With a composite literal, you can't, but you can populate it with initial content.

I usually prefer to one that requires less characters and is the simplest.
When I see `make` I start to think something special happens here.

But yes, all of the above are not strong enough reasons to avoid `make`.